src/main/java/edu/hawaii/its/groupings/access/User.java

```diff
@@ -45,6 +45,10 @@ public String getName() {
         return attributes.getValue("cn");
     }
 
+    public String getGivenName() {
+        return attributes.getValue("givenName");
+    }
+
```

Comment:
Does getName() in User return the person's full name and getGivenName() return the first name? If so, can you change getGivenName() to getFirstName() to make it more clear.

Reply:
This one is on me. I was torn about this decision and suggested to Carol that we match up the getter with the underlying attribute in LDAP. I'm not sure what got into the LDAP people to name the attribute the way they did. Whatever you decide is good, I think.

Reply:
P.S. A lot of people (but not all) in the programming world have decided that firstname and/or lastname are considered one word, so need for uppercase on the letter N. If you go in for that line of thinking, of course.

Reply:
In that case, I'll keep it as is.